clerk/organizations.go

```diff
@@ -214,3 +214,37 @@ func (s *OrganizationsService) ListMemberships(params ListOrganizationMembership
 	}
 	return membershipsResponse, nil
 }
+
+type CreateOrganizationMembershipParams struct {
+	OrganizationID string
+	UserID         string `json:"user_id" form:"user_id"`
```

Comment:
No need to use a form tag

```suggestion
	UserID         string `json:"user_id"`
```